ros2_control_demo_hardware/include/ros2_control_demo_hardware/diffbot_system.hpp

```diff
@@ -34,7 +34,7 @@ class DiffBotSystemHardware
 : public hardware_interface::BaseInterface<hardware_interface::SystemInterface>
 {
 public:
-  RCLCPP_SHARED_PTR_DEFINITIONS(DiffBotSystemHardware);
+  RCLCPP_SHARED_PTR_DEFINITIONS(DiffBotSystemHardware)
```

Comment:
Revert this change because it fails on our liners.

```suggestion
  RCLCPP_SHARED_PTR_DEFINITIONS(DiffBotSystemHardware);
```